src/dune_rules/coq_rules.ml

```diff
@@ -376,6 +376,7 @@ let setup_coqdep_rule ~sctx ~loc (cctx : _ Context.t) ~source_rule coq_module =
   let source = Coq_module.source coq_module in
   let file_flags =
     let file_flags = Context.coqc_file_flags cctx in
+    (if cctx.buildable.stdlib then [] else [Command.Args.A "-boot"]) @
```

Comment:
Maybe this could go inside `coqc_file_flags` ?

Reply:
Isn't it the case that `-noinit` should be included for `coqdep`? So there are separate code-paths for `coqdep` and `coqc`, which means that this cannot go into `coqc_file_flags`?